src/razor/src/diagnostics/reportIssueCreator.ts

```diff
@@ -160,12 +189,15 @@ ${extensionTable}
 
     // Protected for testing
     protected async getProjectedCSharp(razorDocument: IRazorDocument) {
-        let csharpContent = `////////////////////// Projected CSharp as seen by extension ///////////////////////
+        const projectedCSharpHeaderFooter = l10n.t('CSharp as seen by extension');
```

Comment:
This is missing the word "Projected", but perhaps thats intentional?

Reply:
Restored